habitat_sim/robots/mobile_manipulator.py

```diff
@@ -339,6 +363,9 @@ def arm_joint_pos(self) -> np.ndarray:
     @arm_joint_pos.setter
     def arm_joint_pos(self, ctrl: List[float]):
         """Kinematically sets the arm joints and sets the motors to target."""
+        if len(ctrl) != len(self.params.arm_joints):
+            raise ValueError("Control dimension does not match joint dimension")
```

Comment:
Eventually this will need to be more complex to handle multi-dof spherical joints. Other things in the wrapper will need to be refactored for this also, so fine for now to assume single dof joints.